src/config/mongodb.ts

```
@@ -6,15 +6,30 @@ let client: MongoClient | null = null;
     client = new MongoClient(config.mongodbUri);
     await client.connect();
     db = client.db();
+
     logger.info("MongoDB connected successfully");
+
+    const collection = db.collection("cache");
+
+    await Promise.all([
+      collection.createIndex(
+        { key: 1, expiresAt: 1 },
+        { name: "idx_key_expiresAt" }
```

ESLint

[error] Insert `,`

(prettier/prettier)